spec/lucky/file_response_spec.cr

```diff
@@ -0,0 +1,101 @@
+require "../spec_helper"
+
+include ContextHelper
+
```

Comment:
I think it might be nice to have one integration type spec. You can see an example here: https://github.com/luckyframework/lucky/blob/500e4bc28786722700672c398b768b3937b204cd/spec/lucky/action_rendering_spec.cr. I'm imaging it would have a describe for "sending files"

Reply:
Agreed. Workin on it!